sdk/tests/conformance2/textures/misc/tex-mipmap-levels.html

```diff
@@ -189,6 +211,30 @@
       wtu.glErrorShouldBe(gl, gl.NO_ERROR, "generateMipmap should succeed");
   }
   gl.deleteTexture(tex);
+
+  // Test textureSize should work correctly with non-zero base level
+  var program = wtu.setupProgram(
+      gl, ['vshader_texsize', 'fshader_texsize'], ['vPosition'], [0]);
+
+  gl.uniform1i(gl.getUniformLocation(program, "tex"), 0);
+  gl.uniform4f(gl.getUniformLocation(program, "uMult"), 1, 1, 1, 1);
+  gl.uniform1i(gl.getUniformLocation(program, "lod"), 1);
+  gl.uniform2i(gl.getUniformLocation(program, "texSize"), 7, 4);
+  tex = gl.createTexture();
```

Comment:
Right, but the base level is being set to 1 below, so the call to textureSize for level 1 actually fetches lod 1+1 = 2, which would be (7, 4) -- if I'm understanding the test correctly.